app/controllers/application_controller.rb

```diff
@@ -1,6 +1,8 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery
 
+  before_action :set_raven_context
```

Comment:
Nice, i have to say I haven't looked at Ravens context features. I tend to plug it off in development / test and only leave it for staging/production.